tremor-erl/eqc/offramp_eqc.erl

```diff
@@ -45,7 +45,35 @@ id() ->
     ?SUCHTHAT(Id, ?LET(Id, list(choose($a, $z)), list_to_binary(Id) ), byte_size(Id) > 0).
 
 offramp(#state{root = Root, schema = Schema}) ->
-    ?LET(V, jsongen:json(Schema, [{root, Root}, {depth, 3}]), tremor_http:decode(list_to_binary(jsg_json:encode(V)))).
+    ?LET(V, jsongen:json(Schema, [{root, Root}, {depth, 3}]), offramp_prepare(V)).
+
+offramp_prepare(Offramp) ->
+    Prepared = tremor_http:decode(list_to_binary(jsg_json:encode(Offramp))),
+    offramp_add_valid_config(Prepared).
+
+%% add a valid config - this is to cumbersome to express in openapi.yaml
```

Comment:
perhaps nothing for today but it'd be nice but we should probably ticket that we make generators for configs.